gulpfile.js

```diff
@@ -173,10 +179,11 @@ gulp.task('package:gem', () => {
 
 gulp.task('package:gem:prepare', () => {
   gulp.src(paths.bundleCss + '**/*').pipe(gulp.dest(paths.gemCss))
+  gulp.src(paths.bundleImg + '**/*').pipe(gulp.dest(paths.gemImg))
   gulp.src(paths.bundleScss + '**/*').pipe(gulp.dest(paths.gemScss))
   gulp.src(paths.bundleJs + '**/*').pipe(gulp.dest(paths.gemJs))
   gulp.src(paths.bundleTemplates + '**/*').pipe(gulp.dest(paths.gemTemplates))
-  return gulp.src(`lib/packaging/gem/${packageJson.name}.gemspec`).pipe(gulp.dest(paths.gem))
```

Comment:
Can’t remember exactly why the `return` was here, but I thought it broken if we didn’t have it?